docs/stage/spider-data-catalog.md

```diff
@@ -17,3 +17,22 @@ Any errors encountered during this stage will be stored in the `spider_data_cata
 * `error` - What the error was
 * `found_via` - How we got to this URL. Which data catalogs did we go throught to find this URL?
 * `error_at` - What date and time the error occurred
+
+## Clear out work already done (Database storage)
+
+In effect, this really counts as deleting the whole database, recreating and starting again. 
+You may find it faster to just do that.
+
+But you can clear it out without doing that if you need to.
```

Comment:
This is a useful thing to know, but the 'if' feels like something a reader might not be able to know. 

How about "this removes all data relating to this application from the database. If you've not got any other data in the database and you're comfortable removing and recreating the database, then that's likely to be quicker. If not, however, then proceed." 

Or something like that!